Experimental/README.md

```diff
@@ -0,0 +1,27 @@
+# ‼️Experimental Samples‼️
+
+Experimental samples are new samples or experimental modifications made to mature samples. Unity does not support Experimental samples because they are in the early stages of development, address currently unsupported use cases, or rely on experimental technologies.
+
+Experimental samples can go through many changes before they are ready to be released and labelled as _Production-Ready_. They might pass the verification requirements; however, they could be deprecated. There is no guarantee for future support, so you should not use experimental samples in production.
+
+## Goal and Purpose
+
+The goal of this folder is to aggregate ideas and small projects in development to accelerate community access. They are made available to foster discussion and gauge interest in particular areas. 
+
+**These samples come with no quality guarantee and are _not_ recommended for production use.**
+
+This means:
+- They are unsupported and may be changed or removed at any time. 
+- They are not rigorously tested against Unity Editor versions, platforms, or devices.
+- They are not maintained to the same quality or standards as those labelled as _Production-Ready_.
```

Comment:
```suggestion
- They are not **maintained** to the same quality or standards as those labelled as _Production-Ready_.
```
I read this very quickly and didn't catch the maintained, and thought the quality and standards would not be upheld. This could draw eyes quickly to the maintainability of the samples.

Reply:
mmm your first read was right, there's no guarantee on the quality and standards of the sample itself, even less on its maintainability. Unless I misunderstand what you mean?

Reply:
Aligned offline! This reads well.